### ISSUE TITLE
cmake install error - template_user_style.xml no such file or directory

### ISSUE THREAD

**Issue by cmorgenstern** [giuspen/cherrytree#1521]
**Version, Operative system**
`cherrytree-git 0.99.30.r21.g724766d8`
Arch Linux `5.10.16-arch1-1 x86_64 GNU/Linux`

**Describe the bug**
cmake installation is failing due to a missing XML template:
```
CMake Error at cmake_install.cmake:77 (file):
  file INSTALL cannot find
  "/build/cherrytree-git/src/cherrytree/data/template_user_style.xml": No
  such file or directory.


make: *** [Makefile:138: install] Error 1
make: Leaving directory '/build/cherrytree-git/src/cherrytree/build'
==> ERROR: A failure occurred in package().
    Aborting...
```
From my build tests, I have determined that this issue was introduced in [this commit](https://github.com/giuspen/cherrytree/commit/3299540a58712b0e6836b9fa1abcc7d76b7a9653). It looks like this file was re-named to `user-style.xml`

**To Reproduce**
1. Build cherrytree from commit [`3299540`](https://github.com/giuspen/cherrytree/commit/3299540a58712b0e6836b9fa1abcc7d76b7a9653) or later
2. Install fails with the error above

**Comment by giuspen** [comment]
Thanks I'll sort it out today it's nice to see somebody is also using the latest from git

**Comment by cmorgenstern** [comment]
Confirmed fixed when building from commit [`f3a9110`](https://github.com/giuspen/cherrytree/commit/f3a91106b85fcdc2c4750f97f35d148767d0884a) - thank you!

I maintain the Arch Linux User Repository (AUR) package [cherrytree-git](https://aur.archlinux.org/packages/cherrytree-git/) that builds from the latest commit, so I'll try and report any breakage in a timely manner.

**Comment by giuspen** [comment]
Excellent thank you ;)
